setup.py

```diff
@@ -35,15 +36,22 @@ def find_version(*paths):
     packages=setuptools.find_packages(where=".", include="deltacat*"),
     install_requires=[
         # any changes here should also be reflected in requirements.txt
-        "s3fs >= 2021.7.0",
-        "tenacity >= 8.0.1",
-        "ray == 1.10.0",
+        "s3fs == 2022.1.0",
+        "memray == 1.3.1",
+        "tenacity == 8.0.1",
+        "ray[default] == 2.1.0",
         "pandas >= 1.3.1",
-        "pyarrow >= 5.0.0",
+        "pyarrow == 8.0.0",
+        "pydantic == 1.10.2",
         "numpy >= 1.21.1",
-        "boto3 == 1.20.40"
+        "boto3 == 1.20.24",
+        "typing-extensions == 4.4.0",
+        "binpacking == 1.5.2"
```

Comment:
It seems like this is only used for tests - can we move it into a separate requirements.txt under scalability tests?

Reply:
yes, binpacking can be removed for now.